packages/fluent-editor/src/config/types/editor-config.interface.ts

```diff
@@ -22,6 +22,7 @@ export interface IEditorConfig extends QuillOptions {
     maxSize?: number
     success?: (file: File) => void
     fail?: (file: File) => void
+    multi?: boolean
```

Comment:
@qwangry 这个名字建议用完整单词：multiple
```suggestion
    multiple?: boolean
```

Reply:
好滴